expr/func_linearregression.go

```diff
@@ -0,0 +1,144 @@
+package expr
+
+import (
+	"fmt"
+	"math"
+	"time"
+
+	"github.com/grafana/metrictank/api/models"
+	"github.com/grafana/metrictank/api/tz"
+	"github.com/grafana/metrictank/mdata"
+	"github.com/grafana/metrictank/schema"
+)
+
+type FuncLinearRegression struct {
+	in GraphiteFunc
+
+	startSourceAt string // at(1) format
```

Comment:
Not sure we should reference `at(1)` here. Better to say `Render Time format` probabky.

Reply:
Technically it's already referencing `at(1)` here (the `At` suffixes which come from the argument name).

Reply:
Actually I guess it's pretty ambiguous as to whether they mean `at(1)` format or "start the source _at_ this time"